Automattic/wp-calypso
# Site Settings: Rename traffic to settings-traffic

This PR renames the `traffic` directory in site-settings to `settings-traffic`. This was recommended by @mtias a while ago - see https://github.com/Automattic/wp-calypso/pull/12496#discussion_r107907434.

To test:
* Checkout this branch
* Go to `/settings/traffic/$site`
* Verify the Traffic section loads correctly with no regressions.
